src/logcollector/read_win_event_channel.c

```diff
@@ -240,10 +132,19 @@ char *get_message(EVT_HANDLE evt, LPCWSTR provider_name, DWORD flags)
                                          0,
                                          0);
     if (publisher == NULL) {
+        LSTATUS err = GetLastError();
```

Comment:
What is the motivation for this change? Could you describe it and show a log sample?

Reply:
As the message is obtained from the provider name included in the event, it could be useful to show which is the value for this field, as well as converting to string the error code returned. Here is an example:

![image (5)](https://user-images.githubusercontent.com/14914658/58458272-135cc580-8129-11e9-8703-2e1ff6021bf1.png)

Reply:
Ok, I agree with you that now is more descriptive. However, its level should be changed to `debug1` due to we have another error message more user-friendly when this error happens. It would look like:

```
2019/05/28 11:08:44 ossec-agent[928] read_win_event_channel.c:246 at get_message(): DEBUG: Could not EvtOpenPublisherMetadata() with flags (1) which returned (1717): The interface is unknown. 
2019/05/28 11:08:44 ossec-agent[928] read_win_event_channel.c:522 at send_channel_event(): ERROR: Could not get message for (System), provider (Microsoft-Windows-WinRM)
```

The second error would be the one shown by default which is less technical for the user.

Reply:
Updated [b1b0edf](https://github.com/wazuh/wazuh/pull/3393/commits/b1b0edfd12f0b63ae59e5563dfd18be2eb049663)